pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>org.sagebionetworks.bridge</groupId>
     <artifactId>sdk-group</artifactId>
     <!-- Please use the maven versions plugin to manage this, e.g.`mvn versions:set -DnewVersion=0.8.1`-->
-    <version>0.19.25</version>
+    <version>0.19.26</version>
```

Comment:
You should bump this to 0.19.27, otherwise it'll conflict with the other pull request and may cause weird build issues.

Reply:
Moved to 0.19.27 and I will merge after assessment resources.